README.md

```diff
@@ -538,6 +538,12 @@ You may also get the name of the current package through the environment variabl
 $ lerna exec -- npm view \$LERNA_PACKAGE_NAME
 ```
 
+You may also run a script located in the root dir, in a complicated dir structure through the environment variable `LERNA_ROOT_PATH`:  
+
+```sh
+$ cross-env lerna exec node $LERNA_ROOT_PATH/scripts/some-script.js
```

Comment:
I'm not sure why `cross-env` is necessary in this example? Also, the env var needs to be escaped (`\$LERNA_ROOT_PATH`) to be resolved properly, as in the previous `npm view` example.